dockerfile

```diff
@@ -0,0 +1,26 @@
+# pull official base image
+FROM node:14.17-alpine AS builder
+
+# set working directory
+WORKDIR /parity-bridges-ui
+
+RUN apk update
+RUN apk add git
+
+# add `/parity-bridges-ui/node_modules/.bin` to $PATH
+ENV PATH /parity-bridges-ui/node_modules/.bin:$PATH
```

Comment:
Why is this needed exactly? I don't think we us anything specific from that directory, do we?

Reply:
I think that is stray from testing times